ivy/data_classes/container/statistical.py

```diff
@@ -79,9 +79,11 @@ def min(
         """
         return self.cont_handle_inplace(
             self.cont_map(
-                lambda x_, _: ivy.min(x_, axis=axis, keepdims=keepdims)
-                if ivy.is_array(x_)
-                else x_,
+                lambda x_, _: (
+                    ivy.min(x_, axis=axis, keepdims=keepdims)
+                    if ivy.is_array(x_)
+                    else x_
+                ),
```

Comment:
This is improved parentheses managment, black wraps `x if c else y` with parentheses, and it looks like we have a lot of those in our code. I guess it's some improved readablity (at least for me).